readme/Welcome.md

```diff
@@ -0,0 +1,31 @@
+# Welcome to HotSoup!
+
+HotSoup is a free, open source kitchen soup location app! It is free and downloadable across both iOS and Android platforms. It helps users to find local hot soup kitchens with 1-3 clicks, by allowing the user to either search by their location or to enter an address. The goal is to help people who are food insecure find their next free, hot meal easier and faster. With accessibility and ease of use in mind, we aim to help as many folks get fed as possible easier and with comapssion. HotSOup is available as a **mobile** format with **desktop/web** coming soon!
+
+This project is current run exclusively by a wonderful crew of 100% volunteers, please take a look at our amazing Discord server where all the teams are working together to coordinate the features and deliverables - you might be amazed to see the level of work these folks are producing, especially since a lot of us are new to the software industry!!!
+
+https://discord.gg/3H9CFN3Wqd
+
+## HotSoup is divided into 2 parts
+
+HotSoup has 4 main UX types:
+
+- **Screens** that the user can navigate to freely or is navigated to automatically depending on which user flow the user is currently partaking in.
+
+- **Tabs and Stack Navigators** to allow the user to enter/exit/control their desired user-flow movement within the app to customize their user experience at any time.
+
+- **Integrated Map API** to render the search results in a commonly used and easy to understand format to allow the user to see where on the map the results are. The map is interactive, can be clicked on to generate directions to a map location, pinchable to zoom in/out, and also has all of the other standard types of interactivity integrated map APIs have.
```

Comment:
"...easy to understand formating, ALLOWING the user to see where THE RESULTS ARE on the map."

"The map is interactive: it can be clicked to generate directions to a map location, pinchable to zoom in/out, and has the standard types of interactivity that integrated map APIs have."

Reply:
done and done, thanks!